doc/sql.extensions/README.data_types

```diff
@@ -200,28 +200,43 @@ Enhancement in precision of calculations with NUMERIC/DECIMAL (FB 4.0)
 --------------
 
   Function:
-    Maximum precision of NUMERIC and DECIMAL data types is increased to 34 digits.
+    Maximum precision of NUMERIC and DECIMAL data types is increased to 38 digits.
 
   Author:
     Alex Peshkoff <peshkoff@mail.ru>
 
   Syntax rules:
     NUMERIC ( P {, N} )
     DECIMAL ( P {, N} )
-		where P is precision (P <= 34, was limited prior with 18 digits) and N is optional number
+		where P is precision (P <= 38, was limited prior with 18 digits) and N is optional number
 		of digits after decimal separator (as before).
 
   Storage:
-    128-bit, format according to IEEE 754.
+    128-bit signed integer.
 
   Example(s):
     1. DECLARE VARIABLE VAR1 DECIMAL(25);
-    2. CREATE TABLE TABLE1 (FIELD1 NUMERIC(34, 17));
+    2. CREATE TABLE TABLE1 (FIELD1 NUMERIC(38, 19));
 
   Note(s):
     Numerics with precision less than 19 digits use SMALLINT, INTEGER, BIGINT or DOUBLE PRECISION
     as base datatype depending upon number of digits and dialect. When precision is between 19 and
-    34 digits DECFLOAT(34) is used for it. Actual precision is always increased to 34 digits. For
-    complex calculations such digits are casted (internally, in trivial way) to DECFLOAT(34) and
-    the result of various math (log, exp, etc.) and aggregate functions using high precision
-    numeric argument is DECFLOAT(34).
+    38 digits 128-bit integer is used for it. Actual precision is always increased to 38 digits.
+    For complex calculations such digits are casted (internally) to DECFLOAT(34) and the result of
+    various math (log, exp, etc.) and aggregate functions using high precision numeric argument is
+    DECFLOAT(34).
+
+	SET INT128 BIND <bind-type> - controls how are INT128 values represented in outer
+	world (i.e. in messages or in XSQLDA). Valid binding types are: NATIVE (use 128-bit
+	binary representation), CHAR/CHARACTER (use ASCII string), DOUBLE PRECISION (use
+	8-byte FP representation - same as used for DOUBLE PRECISION fields) or BIGINT
+	with possible comma-separated SCALE clause (i.e. 'BIGINT, 3'). Various bindings
+	are useful if one plans to use DECFLOAT values with some old client not supporting
```

Comment:
Mark, I don’t understand why you need backward compatibility with previous versions 4.0. These versions were never released; they could not be used in industrial operation. Those who used are to blame for themselves. Everywhere it was written that the implementation in Alpha and Beta is not final.

Reply:
I assume this was a reply to my general review comment and not on this specific review comment?

Breaking changes after a beta release should be explicitly documented somewhere. I don't think expect full compatibility, but I do think this should either produce errors (which I suspect won't happen, you probably just get wrong values), or some form of fixing things through a backup and restore should be available. This would probably be appreciated by people who are actively testing things.